docs/src/dedicated-gen-3/monitoring.md

```diff
@@ -2,59 +2,53 @@
-This APM relies on [Blackfire.io](../../increase-observability/integrate-observability/blackfire.md#on-dedicated-gen-2-infrastructure),
-which uniquely combines monitoring, profiling, and performance testing technologies.
+As the official, in-house Platform.sh observability tool, [Blackfire](../../increase-observability/integrate-observability/blackfire.md) provides unparalleled monitoring, profiling, and performance testing technologies.
+Using Blackfire on Platform.sh enhances your experience
+and allows you to enjoy greater support as well as unique upcoming features.
 
-The Observability Suite is currently available only to Enterprise, Elite, and Fleet customers.
-The Observability Suite package includes all Blackfire features, support, and usage that scales with your needs.
-If you are a Enterprise, Elite, or Fleet customer and want to add the Observability Suite, [contact Sales](https://platform.sh/contact/).
+You can subscribe to Blackfire in two different ways:
 
-All customers can also [subscribe to Blackfire separately](https://www.blackfire.io/pricing).
-This lets you benefit from all of its features on Platform.sh.
-When purchased separately, features and usage may cost more than the equivalent bundled in the Observability Suite.
+- As an Enterprise or Elite customer,
+  you can sign up for the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
+  which offers application performance monitoring by Blackfire packaged with infrastructure monitoring.
+  The Observability suite includes all Blackfire features, support, and usage that scales with your needs.
+  To subscribe to the Observability Suite, [contact Sales](https://platform.sh/contact/).
 
-By using Blackfire on Platform.sh, you benefit from an enhanced experience and support as well as upcoming unique features.
-You can contact your sales representative to get a quote for whatever size cluster is running your application.
+- All customers can also [subscribe to Blackfire](https://www.blackfire.io/pricing) separately.
+  To get a quote based on the size of your cluster, [contact Sales](https://platform.sh/contact/).
+  Note that if you subscribe to Blackfire separately,
+  features and usage may cost more than the equivalent bundled in the Observability Suite.
 
-Platform.sh also [supports New Relic APM](../../increase-observability/integrate-observability/new-relic/_index.md#on-a-dedicated-gen-2-cluster).
-After you have signed up with New Relic and gotten your license key,
-open a support ticket so that it can be installed on your project.
-New Relic infrastructure monitoring isn't supported.
+Platform.sh also supports third-party observability services
+such as [New Relic](../increase-observability/integrate-observability/new-relic/_index.md)
+and [Tideways](../increase-observability/integrate-observability/tideways.md).
+These third-party services have their own associated cost,
```

Comment:
It's unclear that the customer will need to get its own license for NR and Tidedays.